docs/roadmap.md

```diff
@@ -37,9 +37,9 @@ We're focused on enabling three main use cases:
 3. **Enabling other frameworks**
     * Providing the native implementation for other frameworks like [React Native](https://github.com/Microsoft/react-native-windows) when running on Windows
 
-The team recently released WinUI 3 Preview 1, which is an early pre-release that includes support for Win32 and UWP apps. Preview 1 is available for anyone to try out, but note that it has limitations and known issues, so it is not equipped for production apps. Get started or read more about Preview 1 here: 
+The team recently released WinUI 3 Preview 2, which is an early pre-release that includes support for Win32 and UWP apps. Preview 21 is available for anyone to try out, but note that it has limitations and known issues, so it is not equipped for production apps. Get started or read more about Preview 2here: 
```

Comment:
Two small typos:

* Preview 21 -> Preview 2
* Preview 2here -> Preview 2 here

Reply:
Wow, I don't know how I submit this PR without catching those. Thanks @Felix-Dev!